qa/restart-upgrade/build.gradle

```diff
@@ -35,6 +35,7 @@ testClusters {
     }
 }
 
+   def versionsBelow2_3 = ["1.", "2.0.", "2.1.", "2.2."]
```

Comment:
I see that we have added test for restart upgrades here. Do we have plans to add test for rolling upgrades too in follow up PR?

Reply:
rolling upgrade will only hit native code path after 2.17 since we just allow 1 version upgrade so its moot unless I am missing something

Reply:
Make sense.. i forgot that